frontend/src/contexts/CurrentUserContext.js

```diff
@@ -0,0 +1,4 @@
+import  { createContext } from "react";
+
+export const CurrentUserContext = createContext();
+export const isLoggedInContext = createContext();
```

Comment:
`isLoggedInContext` should be in **UpperCase** becuase any context variable acts as a component